cmd/kubeadm/app/cmd/options/generic.go

```diff
@@ -100,3 +100,19 @@ func AddPatchesFlag(fs *pflag.FlagSet, patchesDir *string) {
 		`which patches are applied first alpha-numerically.`,
 	)
 }
+
+// SetGlobalFlags set global flags
+func SetGlobalFlags(fs *pflag.FlagSet) {
```

Comment:
i don't see this function being called.
it duplicates cmd/kubeadm/app/kubeadm.go#Run()

Reply:
Oh... This method is supposed to be added by using setGlobalFlags to set the klog parameter. Later, I did not want to change so much code, so I restored it again. However, I forgot to delete this part of code.

It's been delete.